Bottom: 7af864c41d576a7c3b80ae4b062181c832dcc382
Top:    9769845225f90673c1d4d72869950a5e27cc374d
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-02-21 16:44:19 +0800

Refresh of virtio-net-optimize-small

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index cdff93c..eccde74 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -360,6 +360,7 @@ static bool virtnet_xdp_xmit(struct virtnet_info *vi,
 	/* Free up any pending old buffers before queueing new ones. */
 	while ((xdp_sent = virtqueue_get_buf(sq->vq, &len)) != NULL) {
 		struct page *sent_page = virt_to_head_page(xdp_sent);
+
 		put_page(sent_page);
 	}
